README.md

```diff
@@ -1,81 +1,119 @@
+
 ```vim
 lua << EOF
 require'lspconfig'.pyright.setup{}
 eof
 ```
-3. Create a project, this project must contain a file matching the root directory trigger. See [Automatically launching language servers](#Automatically-launching-language-servers) for additional info.
+
+1. Create a new project.
+   To work with LSP your project will need to have one of several root-directory marker files or folders.
+   For instance, a `.git` folder is considered a root-directory marker.
+   See [Automatically launching language servers](#Automatically-launching-language-servers) for additional info.
+
 ```bash
 mkdir test_python_project
 cd test_python_project
 git init
 touch main.py
 ```
-4. Launch neovim, the language server will now be attached and providing diagnostics (see `:LspInfo`)
-`nvim main.py`
-5. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling omnifunc completion
+
+1. Launch Neovim: `nvim main.py`.
+   The language server will now be attached and providing diagnostics (run `:LspInfo` for details).
+1. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling 'omnifunc' (`\<C-x\>\<C-o\>`) completion
 
 ## Automatically launching language servers
 
-In order to automatically launch a language server, lspconfig searches up the directory tree from your current buffer to find a file matching the `root_dir` pattern defined in each server's configuration.  For [pyright](CONFIG.md#pyright), this is any directory containing ".git", "setup.py",  "setup.cfg", "pyproject.toml", or "requirements.txt").
+In order to automatically launch a language server, lspconfig searches up the directory tree from your current buffer to find a file matching the `root_dir` pattern defined in each server's configuration.
+For [pyright](CONFIG.md#pyright), this is any directory containing one of:
+
+- `.git`,
+- `setup.py`,
+- `setup.cfg`,
+- `pyproject.toml`, or
+- `requirements.txt`.
 
-Language servers require each project to have a `root` in order to provide completion and search across symbols that may not be defined in your current file, and to avoid having to index your entire filesystem on each startup.
+Language servers require each project to have a `root` in order to provide completion and search across symbols that may not be defined in your current file; it also avoids having to index your entire filesystem on each startup.
 
 ## Enabling additional language servers
 
-Enabling most language servers is as easy as installing the language server, ensuring it is on your PATH, and adding the following to your config:
+Most language servers can be installed in less than a minute.
+Enabling a language server means:
+
+1. Installing the language server binary: `npm install global pyright`;
+1. Ensuring the language server can be found on your `PATH`: typing `pyright --version` succeeds;
```

Comment:
This whole thing is kinda a dupe of the quickstart

Reply:
Yes, I think so too, could probably be removed completely.

Reply:
I think we need to include is as people have shown that they do not generalize from the quickstart to installing another language server, I pick rust analyzer due to popularity, but tsserver is probably the most common one